R/zzz.R

```diff
@@ -13,46 +13,41 @@
 
 #' Set package options with regard to computation times
 #' 
-#' These functions set options of the bfast and strucchangeRcpp package to enable
-#' faster computations. The fast options should return equal results but
-#' require the \code{strucchangeRcpp} package, which includes
-#' matrix-based functions. Notice that only some functions of the \code{bfast}
-#' package make use of these options.
+
+#' These functions set options of the bfast and strucchangeRcpp packages to enable
+#' faster computations. By default (\code{set_default_options}), these optimizations are 
+#' enabled. Notice that only some functions of the \code{bfast}
+#' package make use of these options. \code{set_fastt_options} is an alias for \code{set_default_options}.
 #' 
 #' @name setoptions
-#' @aliases set_fast_options set_default_options
+#' @aliases set_default_options set_fallback_options set_fast_options
 #' @return A list of modified options and their new values.
-#' @author Marius Appel
 #' @examples
 #' 
 #' 
 #' # run bfastmonitor with different options and compare computation times
 #' library(zoo)
 #' NDVIa <- as.ts(zoo(som$NDVI.a, som$Time))
 #' 
-#' set_fast_options()
+#' set_default_options()
 #' system.time(replicate(100,  bfastmonitor(NDVIa, start = c(2010, 13))))
 #' 
-#' set_default_options()
+#' set_fallback_options()
 #' system.time(replicate(100,  bfastmonitor(NDVIa, start = c(2010, 13))))
 #' 
-#' @export set_fast_options set_default_options
-set_fast_options <- function() {
-  if (!requireNamespace("strucchangeRcpp", quietly = TRUE)) {
-    warning("package strucchangeRcpp required for enabling fast options; using default implementation of strucchange")
-  }
-  else {
-    return(options(strucchange.use_armadillo=TRUE, 
-                   bfast.prefer_matrix_methods=TRUE,
-                   bfast.use_bfastts_modifications=TRUE))
-  }
+#' @export set_fast_options set_default_options set_fallback_options
+set_default_options <- function() {
+  return(options(strucchange.use_armadillo=TRUE, 
+                 bfast.prefer_matrix_methods=TRUE,
+                 bfast.use_bfastts_modifications=TRUE))
 }
 
+set_fast_options <- function() {
+  set_default_options()
+}
```

Comment:
Better not to create needless nesting:
```suggestion
set_fast_options <- set_default_options
```